addon/components/power-select.js

```diff
@@ -659,7 +659,9 @@ export default Component.extend({
   _handleKeySpace(e) {
     let publicAPI = this.get('publicAPI');
     if (publicAPI.isOpen && publicAPI.highlighted !== undefined) {
-      e.preventDefault(); // Prevents scrolling of the page.
+      if (e.target === document.body) {
+        e.preventDefault(); // Prevents scrolling of the page.
```

Comment:
@cibernox, I see you have covered this `preventDefault()` in your test [keyboard-control-test.js:174](https://github.com/cibernox/ember-power-select/blob/master/tests/integration/components/power-select/keyboard-control-test.js#L174)
I was thinking .. I read somewhere that scrolling happens on the body. I'm not sure if that is correct though. Maybe we can detect if it's an elemnt where you can type something. Then we can actually test it. Otherwise, should we remove the test? This will never be `true` => `e.target === document.body`  in the test I think
```js
if (['TEXTAREA', 'INPUT'].includes(e.target.nodename)) {
  e.preventDefault(); 
}
```
What do you think?

Reply:
Actually .. come to think of it .. we should not handle the spacebar at all when we are in an input field or a text area .. 😅